src/wasm/kotlin/java/arcs/wasm/main.kt

```diff
@@ -0,0 +1,4 @@
+package arcs
+
+fun main() {
```

Comment:
It'd be nice to add some comments here explaining why this is needed and why it's empty.. also, could make it `fun main() = Unit`